Use "command -v" instead of "which" to fix build on Tiger when gtk-doc
is not installed.
https://gitlab.gnome.org/GNOME/glib/merge_requests/277
--- autogen.sh.orig
+++ autogen.sh
@@ -7,8 +7,7 @@ test -n "$srcdir" || srcdir=.
 olddir=`pwd`
 cd "$srcdir"
 
-GTKDOCIZE=$(which gtkdocize 2>/dev/null)
-if test -z $GTKDOCIZE; then
+if ! command -v gtkdocize >/dev/null 2>&1; then
         echo "You don't have gtk-doc installed, and thus won't be able to generate the documentation."
         rm -f gtk-doc.make
         cat > gtk-doc.make <<EOF
@@ -19,8 +18,7 @@ else
         gtkdocize || exit $?
 fi
 
-AUTORECONF=`which autoreconf`
-if test -z $AUTORECONF; then
+if ! command -v autoreconf >/dev/null 2>&1; then
         echo "*** No autoreconf found, please install it ***"
         exit 1
 fi
